lambdas/abandon/src/test/java/uk/gov/di/ipv/cri/kbv/api/handler/AbandonKbvHandlerTest.java

```diff
@@ -115,7 +115,7 @@ void shouldReturnErrorBadRequestWhenItCannotSendAuditEvent() throws SqsException
         doNothing().when(mockSessionService).createAuthorizationCode(mockSessionItem);
         doThrow(SqsException.class).when(mockAuditService).sendAuditEvent(anyString(), any());
 
-        var result = abandonKbvHandler.handleRequest(input, mock(Context.class));
+        abandonKbvHandler.handleRequest(input, mock(Context.class));
```

Comment:
Test says shouldReturnErrorBadRequest.....
so doing this is better  `assertEquals(HttpStatusCode.BAD_REQUEST, result.getStatusCode());`

Reply:
I'll also like to see what kind of event rather than having to look at the code 

`        doThrow(SqsException.class).when(mockAuditService).sendAuditEvent(eq("ABANDONED"), any(AuditEventContext.class));
`

Reply:
That makes sense, I've added that to the unit test as well